src/components/UI/search/Search.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+import { Input, InputGroup, Stack } from '@chakra-ui/react'
+
+import { BORDER_WIDTH } from '../../../constants'
+import { LensIcon } from '../icons';
+
+export const Search: FC = () => {
+  return (
+    <Stack
+      borderBottom={{ base: BORDER_WIDTH, md: 'none' }}
+      borderRight={{ base: 'none', md: BORDER_WIDTH }}
+      borderColor={{ base: 'bg', md: 'primary' }}
+      p={{ base: 8, md: 4 }}
+      _hover={{ base: {bg: 'primary'}, md: {bg: 'none'}}}
+    >
+      <InputGroup>
```

Comment:
we use [less padding for search input](text [should be centered](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=885%3A8258&t=Ic6WpLw62Cylvxti-0))